src/components/ChatList/ChannelItem.style.ts

```diff
@@ -1,22 +1,22 @@
 import {StyleSheet} from 'react-native';
 
 import dimen from '../../utils/dimen';
-import {colors} from '../../utils/colors';
+import {COLORS} from '../../utils/theme';
 
 export const channelItemStyles = StyleSheet.create({
   channelContainer: {
     display: 'flex',
     flexDirection: 'row',
     paddingLeft: dimen.normalizeDimen(20),
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
   },
   contentContainer: {
     flex: 1,
     paddingTop: dimen.normalizeDimen(14),
     paddingBottom: dimen.normalizeDimen(14),
     paddingStart: dimen.normalizeDimen(15),
     paddingEnd: dimen.normalizeDimen(20),
-    borderBottomColor: colors.alto,
+    borderBottomColor: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `borderBottomColor` property in the `contentContainer` style has been updated to use `COLORS.lightgrey` instead of `colors.alto`. Ensure that this color change is intentional and aligns with the desired visual design of the application.
```diff
-    borderBottomColor: colors.alto,
+    borderBottomColor: COLORS.lightgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->